src/app/zones/locales/en-us/index.yaml

```diff
@@ -77,19 +77,19 @@ zones:
     delete_button: 'Delete'
   form:
     exit: 'Exit'
-    title: 'Create & connect Zone'
-    description: ' '
-    facts: []
+    title: 'Create & Connect Zone'
+    description: 'Provision and connect a remote zone to your Kuma Global Control Plane.'
+    facts: ['Observe and manage zone workloads', 'Gain insight into service configuration and stats', 'View and modify mesh policies', '']
```

Comment:
**Issue** (minor): This will render an empty fact item otherwise

```suggestion
    facts: ['Observe and manage zone workloads', 'Gain insight into service configuration and stats', 'View and modify mesh policies']
```